itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/odo/ConnectionODOCommandsTests.java

```diff
@@ -61,12 +61,14 @@ public void setUp() {
 		connection = explorer.getOpenShiftODOConnection();
 	}
 
+	@Ignore
 	@Test
 	public void testListCatalogServices() {
 		connection.listCatalogServices();
 		new WaitUntil(new TerminalContainsText("Operators"), TimePeriod.LONG);
 	}
 
```

Comment:
Why are the tests ignored?

Reply:
Tthe command does not really exist anymore.